main.tf

```diff
@@ -1,3 +1,11 @@
+resource "aws_acm_certificate" "cert" {
```

Comment:
v2.0.0 of `terraform-aws-api-gateway` no longer creates the certificate resource, so we create it here and pass the id (arn) to the module